scripts/updateSitemapJson.ts

```diff
@@ -0,0 +1,64 @@
+import "dotenv/config"
+
+import { Collection } from "../src/Entities"
+import { loadCsvAsCollection } from "./loadCsvAsColleciton"
+
+const AWS = require("aws-sdk")
```

Comment:
I'm not sure why not `import` did not work but `require` did...

Reply:
did you try `import * as AWS from "aws-sdk"`